bundles/org.openhab.binding.resol/src/main/java/org/openhab/binding/resol/handler/ResolBridgeHandler.java

```diff
@@ -0,0 +1,472 @@
+
+                                            thingBuilder.withChannel(channel).withLabel(thing.getLabel());
+
+                                            thingHandler.updateThing(thingBuilder.build());
+                                        }
+
+                                        thingHandler.setChannelValue(enumChannelId, pfv.getEnumVariant().getText(lang));
+                                    }
+                                } else {
+                                    logger.trace("ThingHandler for {} not registered.", thingType);
+                                }
+                            }
+                        }
+                    });
+
+                    // Establish the connection
+                    tcpConnection.connect();
+                    Thread.sleep(1000); // after a reconnect wait 1 sec
+                    isConnected = (tcpConnection.getConnectionState().equals(Connection.ConnectionState.CONNECTED));
+                } catch (IOException e) {
+                    logger.trace("Connection failed", e);
+                    unconnectedReason = e.getMessage();
+                    isConnected = false;
+                } catch (InterruptedException e) {
+                    isConnected = (tcpConnection.getConnectionState().equals(Connection.ConnectionState.CONNECTED));
+                } catch (Exception e) {
+                    isConnected = (tcpConnection.getConnectionState().equals(Connection.ConnectionState.CONNECTED));
+                    unconnectedReason = e.getMessage();
+                }
+                if (!isConnected) {
+                    logger.info("Cannot establish connection to {} ({})", ipAddress, unconnectedReason);
+                } else {
+                    unconnectedReason = "";
+                }
+                updateStatus();
+            }
+        }
+    };
+
+    /* check if the given value is a special one like 888.8 or 999.9 for shortcut or open load on a sensor wire */
+    private boolean isSpecialValue(Double dd) {
+        if ((Math.abs(dd - 888.8) < 1) || (Math.abs(dd - (-888.8)) < 1)) {
+            /* value out of range */
+            return true;
+        }
+        if (Math.abs(dd - 999.9) < 1) {
+            /* sensor not reachable */
+            return true;
+        }
+        return false;
+    }
+
+    private synchronized void startAutomaticRefresh() {
+        if (pollingJob == null || pollingJob.isCancelled()) {
+            pollingJob = scheduler.scheduleWithFixedDelay(pollingRunnable, 0, refreshInterval, TimeUnit.SECONDS);
+        }
+    }
+
+    // Methods for ThingHandler
+    public ThingStatus getStatus() {
+        return getThing().getStatus();
+    }
+
+    // internal Methods
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        // No channels - nothing to do
+    }
+
+    @Override
+    public void initialize() {
+        logger.debug("Initializing Resol bridge handler {}", this.toString());
```

Comment:
Can this message be replaced by using the debugger or by increasing the framework's log level? See point 4 https://www.openhab.org/docs/developer/guidelines.html#f-logging